src/command/execute_program.rs

```diff
@@ -150,13 +150,21 @@ fn build_result_archive(res: ResultId) -> Result<(), std::io::Error> {
         }
     }
 
-    let _ = Command::new("zip")
-        .arg("-0")
+    let path_to_res = format!("./archives/{}/results", res.program_id);
+    let result = format!("{}", res.timestamp);
+    let path_to_log = String::from("../../../data");
+    let log = format!("{}_{}.log", res.program_id, res.timestamp);
```

Comment:
Please use more self-explaining variable names (I know that it wasn't pretty previously either^^)

For strings that do not change, please use something like `const SOME_NAME: &str = ...`. `String::from` creates a heap allocated string which incurs unneeded overhead.